test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_smdataparallel.py

```diff
@@ -70,10 +93,8 @@ def test_smdataparallel_mnist(instance_types, n_virginia_ecr_image, py_version,
     """
     Tests smddprun command via Estimator API distribution parameter
     """
-    _, image_framework_version = get_framework_and_version_from_tag(n_virginia_ecr_image)
-    image_cuda_version = get_cuda_version_from_tag(n_virginia_ecr_image)
-    if Version(image_framework_version) < Version("2.3.1") or image_cuda_version != "cu110":
-        pytest.skip("Data Parallelism is only supported on CUDA 11, and on TensorFlow 2.3.1 or higher")
+    validate_or_skip_smdataparallel(n_virginia_ecr_image)
+    validate_or_skip_smdataparallel_efa(n_virginia_ecr_image)
```

Comment:
So the multi-node smdataparallel test would no longer be usable for TF 2.3 images. It will only be applicable on tests which have EFA support, i.e., TF 2.4 and above.

Is this intended?

Reply:
it is not multinode, it just using two instances one by one. 
There is same non efa test as below which will run for tf2.3 (also it runs for above tf2.3)

https://github.com/aws/deep-learning-containers/pull/1044/files#diff-dd278248d0eb52756b3bf592fca639d25738021b922d8f4e4d4f5d54a52ba216R60